earth_enterprise/src/third_party/defusedxml/SConscript

```diff
@@ -0,0 +1,89 @@
+Import('third_party_env')
+
+defusedxml_version='defusedxml-0.6.0'
+defusedxml_ge_version = defusedxml_version.replace('defusedxml', 'defusedxml-ge')
+
+current_dir = Dir('.').abspath
+build_root = '%s/%s' % (current_dir, defusedxml_version)
+install_root = '%s/install' % current_dir
+install_root_opt = '%s/opt/google' % install_root
+
+defusedxml_source = File('#/../../earth_enterprise/third_party/defusedxml/%s.tar.gz' %
+                       defusedxml_version).abspath
+
+defusedxml_env = third_party_env.DeepCopy()
+
+root_dir = Dir(defusedxml_env.exportdirs['root']).abspath
+
+# [1] Extract defusedxml.
+defusedxml_target = '%s/.extract' % current_dir
+defusedxml_extract = defusedxml_env.Command(
+    defusedxml_target, defusedxml_source,
+    [defusedxml_env.MultiCommand(
+        'mkdir -p %s\n'
+        'cd %s\n'
+        'tar xzf %s\n'
+        'touch %s' % (current_dir, current_dir, defusedxml_source,
+                      defusedxml_target))])
+
+if third_party_env['is_min_ubuntu'] and not third_party_env['native_cc']:
+  env_opt = 'ac_cv_prog_cc_c99=no'
+else:
+  env_opt = ''
+
+# [2] Build defusedxml.
+python_cmd = defusedxml_env['python_bin']
+defusedxml_target = '%s/.build' % current_dir
+defusedxml_build = defusedxml_env.Command(
+    defusedxml_target, defusedxml_extract,
+    [defusedxml_env.MultiCommand(
+        'cd %s\n'
+        '%s%s %s setup.py build\n'
+        'touch %s' % (build_root, defusedxml_env['ENV']['mod_env'],
+                      env_opt, python_cmd, defusedxml_target))])
+
+# [3] Create defusedxml master installer.
+install_root_lib = '%s/lib/python%s/site-packages' % (
+    install_root, defusedxml_env['python_major_version'])
+install_root_doc = '%s/share/doc/packages/%s' % (
+    install_root_opt, defusedxml_ge_version)
+
+defusedxml_target = '%s/.install' % current_dir
+defusedxml_install = defusedxml_env.Command(
+    defusedxml_target, defusedxml_build,
+    [defusedxml_env.MultiCommand('\n'.join([
+        'cd %s' % build_root,
+        'rm -rf %s' % install_root_lib,
+        'rm -rf %s' % install_root_doc,
+        'mkdir -p %s' % install_root_lib,
+        'mkdir -p %s' % install_root_doc,
+        'rsync -pvr build/lib.linux-x86_64-%s/ %s' % (install_root_lib),
```

Comment:
The source directory is still wrong on this line. It should be `rsync -pvr build/lib/defusedxml %s' % (install_root_lib)`

Reply:
Also, don't forget to add the copyright comment at the top of the file (see above).